addons/test_mail_full/tests/test_portal.py

```diff
@@ -145,6 +147,188 @@ def test_portal_share_comment(self):
         self.assertEqual(message.author_id, self.partner_2)
 
 
+@tagged('portal')
+class TestPortalFlow(MailCommon, HttpCase):
+    """Share a link by email to a customer without an account for viewing a record through the portal.
+
+    The tests consist in sending a mail related to a record to a customer and checking that the record can be viewed
+    through the embedded link:
+    - either in the backend if the user is connected and has the right to
+    - or in the portal otherwise
+    """
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.customer = cls.env['res.partner'].create({
+            'country_id': cls.env.ref('base.be').id,
+            'email': 'mdelvaux34@example.com',
+            'lang': 'en_US',
+            'mobile': '+32492131422',
```

Comment:
Just asking, is it a fake number ? I know there are some reserved working numbers for testing (not remembering them, maybe internet could answer this ^^ ) .

Reply:
Yes good remark. I didn’t know it existed but it is logical. I have found on wikipedia a range of number for France of “fictitious number” so I have updated the number to use one of them. (I had used a generator but I don’t know if the number is in use or not.)